linker-provided-symbols.txt

```diff
@@ -0,0 +1,3 @@
+__heap_base
+__heap_end
+__c_longjmp
```

Comment:
Can you open a wasm-ld bug regarding this?   I think these should not be necessary. 

Also, perhaps add a comment here (or in the Makefile) with a link the bug so we know its temporary.

Reply:
done

Reply:
https://github.com/llvm/llvm-project/issues/103592